### PR TITLE
Being able to ordering setenv entries.

### PR DESCRIPTION
**Pull request by icalvete**
Sometimes, we need choose the order of the params.

### CODE REVIEW

File: manifests/setenv/entry.pp [hunk 1]
```diff
@@ -18,6 +19,7 @@
   $quote_char  = undef,
   # Deprecated
   $base_path   = '',
+  $order       = 10
```
Comment (mhaskel) [inline manifests/setenv/entry.pp]: Could you add a trailing comma here?

File: manifests/setenv/entry.pp [hunk 2]
```diff
@@ -45,5 +47,6 @@
     ensure  => $ensure,
     target  => $_config_file,
     content => inline_template('export <%= @param %>=<%= @_quote_char %><%= Array(@value).join(" ") %><%= @_quote_char %>'),
+    order   => $order
```
Comment (mhaskel) [inline manifests/setenv/entry.pp]: trailing comma here

**Comment by mhaskel** [comment]
Hi @icalvete,

Thanks for the contribution, and sorry it took so long for us to review this. This looks mostly good, but could you also add the documentation for the new parameter in the README, and update the unit tests at https://github.com/puppetlabs/puppetlabs-tomcat/blob/master/spec/defines/setenv/entry_spec.rb to exercise the new parameter? Thanks again, and let us know if you need any help with the updates!

**Comment by icalvete** [comment]
Hi @mhaskel,

Learn rspec-puppet has been in my TODO for long time.  So thanks you for pushing me to this task.

**Comment by mhaskel** [comment]
@icalvete thanks, looks good! This should be rebased against master and squashed to a single commit, but once that's done it will be ready for merge.

**Comment by mhaskel** [comment]
Great, thank you @icalvete !

**Pull request merged**